packages/ui/src/components/NumberInput/NumberInput.test.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { lgStyleProps, mdStyleProps, NumberInput, smStyleProps } from './NumberInput';
+
+describe('NumberInput', () => {
+  it('renders correctly in not disabled mode', () => {
+    render(<NumberInput size="lg" aria-label="inputNumber" />);
+
+    const numberInput = screen.getByLabelText('inputNumber');
```

Comment:
(Minor)
You could make function for this instead of calling screen method every time
const getElement = () => screen.getByLabelText('inputNumber')

Reply:
It can be done but only if we move the entire `render` method to that function or if the function takes the `ariaLabel` parameter. Otherwise, the function you suggest will depend on the aria-label value that is outside of the scope of that function.
As you write it's a minor thing so in my opinion it could be addressed but doesn't have to.